about.html

```diff
@@ -0,0 +1,127 @@
+        <div class="contact-me">
+            <p>Please contact us per Email for any further questions about GDC 2022!</p>
+            <a href="mailto:gdcregistration@informa.com">gdcregistration@informa.com</a>
+        </div>
+      </div>
+    </header>
+    <!-- logo info section -->
+    <section class="logo-info">
+        <h2 class="logo-title">GDC 2022 Logo</h2>
+        <div class="logo-text">
+          <p>The Game Developers Conference began as an informal gathering of 25 developers in the living room of game designer Chris Crawford.</p>
+          <p class="hidden-text">In 2016, GDC commemorated the 30th edition of the conference. Year after year, GDC continues to define market innovations and provide insight into the future of games.</p>
+        </div>
+        <img src="/assets/frame-logo.png" alt="logo">
+    </section>
+    <!-- past conference -->
+    <section class="past">
+      <h2 class="logo-title">See the past Game Developers Conference</h2>
+      <p>Take a look at the last two conferences that took place in San Francisco, CA USA</p>
+      <div class="past-pic">
+        <div class="pic-col">
+          <img src="/assets/show1.jpg" alt="show1">
+          <div class="layer">
+            <h3>2019</h3>
+            <h4>GDC 2019 in San Francisco, CA USA</h4>
+          </div>
+        </div>
+        <div class="pic-col">
+          <img src="/assets/show2.jpg" alt="show1">
+          <div class="layer">
+            <h3>2020</h3>
+            <h4>GDC 2020 in San Francisco, CA USA</h4>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- partners section -->
+    <section class="partners">
+      <h2 class="titles">Partners</h2>
+      <div class="list">
+        <img src="/assets/partners/partner1.png" alt="partners">
+        <img src="/assets/partners/partner2.png" alt="partners">
+        <img src="/assets/partners/partner3.png" alt="partners">
+        <img src="/assets/partners/partner4.png" alt="partners">
+        <img src="/assets/partners/partner5.png" alt="partners">
+        <img src="/assets/partners/partner6.png" alt="partners">
+        <img src="/assets/partners/partner7.png" alt="partners">
+        <img src="/assets/partners/partner8.png" alt="partners">
+      </div>
+    </section>
+    <!-- footer section -->
+    <footer class="diff">
+      <img src="/assets/frame-logo.png" class="frame" alt="logo">
+      <div class="info">
+        <p>2022 Game Developers Conference. Some Rights Reserved.</p>
+        <p class="hidden">All contents on this website, unless stated otherwise, is at GDC in USA.
+        For detailed usage, function details and permission conditions, see <a href="https://gdconf.com/">https://gdconf.com/</a></p>
+      </div>
+    </footer>
+    <script src="index.js"></script>
+  </body>
```

Comment:
Kindly use good semantics principles for your HTML code, remember that in the `hx` series, they come in the order of `h1,h2,h3 ...h6` meaning they should be followed in that hierarchy in your project